jsconfig.json

```diff
@@ -2,8 +2,8 @@
 	"compilerOptions": {
 		"baseUrl": ".",
 		"paths": {
-			"@/components/*": ["lib/ui-library/src/components/*"]
+			"@/*": ["lib/ui-library/src/*"]
 		}
 	},
-	"exclude": ["node_modules"]
+	"include": ["lib/ui-library/src/**/*", "lib/pkp/js/**/*.js"]
```

Comment:
Does anything else need to be included here, e.g. the top level `js` directory? This includes the `build.js` and the `pkp.min.js` but there are also a few legacy controller files. My guess is this is fine since this is all related to our legacy use of JS, but just wanted to double check.